routes/aliemcards/messages.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express';
+import { requireAuthentication } from '../../helpers/authentication';
+const router = express.Router();
+const CHANNEL_ID = '**********'; // #aliemcards
```

Comment:
This can be found in the admin menu... It's sort of a pain. I'll add it in after the merge